src/ray/raylet/node_manager.cc

```diff
@@ -201,10 +195,12 @@ NodeManager::NodeManager(boost::asio::io_service &io_service, const NodeID &self
 
   if (new_scheduler_enabled_) {
     SchedulingResources &local_resources = cluster_resource_map_[self_node_id_];
-    new_resource_scheduler_ =
+    auto cluster_resource_scheduler =
         std::shared_ptr<ClusterResourceScheduler>(new ClusterResourceScheduler(
             self_node_id_.Binary(),
             local_resources.GetTotalResources().GetResourceMap()));
+    cluster_resource_scheduler_ = cluster_resource_scheduler;
```

Comment:
> The local variable `cluster_resource_scheduler` is not needed?

The local variable is type of `ClusterResourceScheduler` which will feed to the construct of `ClusterTaskManager`, otherwise we need to cast the `cluster_resource_scheduler_` to `ClusterResourceScheduler`.

Reply:
Okey, I will remove it.